### PR TITLE
docs: Adapt to docusaurus syntax

### PR DESCRIPTION
**Pull request by matevz**

### CODE REVIEW

**Review by kostko — APPROVED** [review]
Looks good!

File: .markdownlint.yml [hunk 1]
```diff
@@ -7,8 +7,7 @@
 default: true
 
 line-length:
-  # Line length checking is not strict by default.
-  strict: true
+  strict: false
```
Comment (tjanez) [inline .markdownlint.yml:10]: Please, revert this one and group links at the bottom of sections/pages and use:
```
<!-- markdownlint-disable line-length -->
  ...
<!-- markdownlint-enable line-length -->
```

File: .changelog/4501.internal.md [hunk 2]
```diff
@@ -0,0 +1 @@
+docs: Migrate to docusaurus
```
Comment (tjanez) [inline .changelog/4501.internal.md:1]: Please, use the `doc` Change Log fragment type and reword it to something like:
"Migrate Docs to Docusaurus"

File: README.md [hunk 3]
```diff
@@ -6,7 +6,6 @@
 [![Docker status][github-docker-badge]][github-docker-link]
 [![Release status][github-release-badge]][github-release-link]
 [![GoDev][godev-badge]][godev-link]
-[![GitBook][gitbook-badge]][gitbook-link]
```
Comment (tjanez) [inline README.md:9]: Can you add a Docusaurus badge that link to our Docs site from the top of the README?

Comment (matevz) [reply README.md:9]: Nice ;)
